src/components/Table/GridTable.tsx

```diff
@@ -284,11 +292,26 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     return rows;
   }, [columns, rows, sorting, sortState]);
 
-  const columnSizes = useMemo(
-    () => calcColumnSizes(columns, style.nestedCards?.firstLastColumnWidth),
-    [columns, style.nestedCards?.firstLastColumnWidth],
+  const [columnSizes, setColumnSizes] = useState<string[]>(
```

Comment:
Can we in-source comment this with like `// Calc our initial/first render sizes where we won't have a width yet`?

Reply:
Also, just super-naive question, but if on the 1st render, we just output a dummy div, and used it to just measure the width we have available, and then immediately re-render, would the user really notice? And would that be simpler than having a dual "we don't know the width" / "actually know we do know the width"? Or is that sort of bookkeeping net/net more complex?

Reply:
Yea, I tried that "dummy div" idea quickly, but was running into errors. I definitely think there is room for improvement to not render the whole table just to get the width.

Reply:
The error happens when using Virtuoso: 
```
Rendered more hooks than during the previous render.
```
Haven't dug in any deeper.